fr/lessons/basics/documentation.md

```diff
@@ -0,0 +1,340 @@
+---
+layout: page
+title: Documentation
+category: basics
+order: 11
+lang: fr
+---
+
+Documenter du code Elixir.
+
+## Table des Matières.
+
+- [Annotations](#annotations)
+  - [La documentation linéaire](#documentation-lineaire)
+  - [Documenter les modules](#documenter-les-modules)
+  - [Documenter les fonctions](#documenter-les-fonctions)
+- [ExDoc](#exdoc)
+  - [Installer](#installer)
+  - [Générer la documentation](#generer-la-documentation)
+- [Les bonnes pratiques](#les-bonnes-pratiques)
+
+
+## Annotations
+
+La quantité de commentaires et les tenants d'une documentation de qualité demeurent un sujet problématique au sein du monde de la programmation.
+Néanmoins, nous pouvons tous convenir que la documentation est importante, autant pour nous-même que pour les personnes amenées à travailler avec
+notre code.
+
+Elixir traite la documentation comme une *citoyenne de premier ordre*, et offre de multiples fonctions pour accéder et générer de la documentation
+pour vos projets. La bibliothèque standard d'Elixir fourni différents attributs pour annoter un code source. Examinons trois d'entre eux : 
+>>>>>>> fr-documentation
+  - `#` - Pour la documentation linéaire.
+  - `@moduledoc` - Pour la documentation à l'échelle d'un module.
+  - `@doc` - Pour la documentation à l'échelle d'une fonction.
+
+### La documentation linéaire
```

Comment:
Because Jekyll skips accented letters when naming the header, you need to add a custom anchor here or the summary's anchor will not work.